src/authors/jennablumenthal.md

```diff
@@ -1,14 +1,19 @@
 ---
 name: Jenna Blumenthal
 username: jennablumenthal
-twitter: https://twitter.com/jennaleeblume
+twitter: https://twitter.com/jennaleeb
 github: https://github.com/mveytsman
```

Comment:
This one is `jennaleeb` (i know, different handles, am i even a real programmer?)